packages/web3-hooks/base/src/useNonFungibleAssets.ts

```diff
@@ -9,7 +9,6 @@ import { useWeb3State } from './useWeb3State.js'
 
 export function useNonFungibleAssets<S extends 'all' | void = void, T extends NetworkPluginID = NetworkPluginID>(
     pluginID?: T,
-    schemaType?: Web3Helper.SchemaTypeScope<S, T>,
```

Comment:
Please keep it for another chain implementation.

Reply:
This hook is already cross chains, not EVM speicified.

Reply:
This parameter is a public shared design. In case we need to use schema type to filter out a specific kind of tokens. Not only this hook but also other NFT-related hooks use this parameter. So keeping it will help align the design with other hooks, even if it's not used here.